spec/takeaway_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'takeaway'
+
+describe TakeAway do
+  let(:takeaway) { TakeAway.new }
+  before do
+    takeaway.order("Pasta")
+    takeaway.order("Pizza")
+    takeaway.order("Burger")
+  end
+
+  it "shows the list of dishes with prices" do
+    expect(takeaway.menu).to be_an_instance_of(Array)
+  end
+
+  it "can add a menu item to the order" do
+    expect(takeaway.order("Pizza")).to include ({ dish: "Pizza", price: 7.49 })
+  end
+
+  it "shows the total amount of items" do
+    expect(takeaway.total_quantity).to eq 3
+  end
+
+  it "shows the total price of the order" do
+    expect(takeaway.total_amount).to eq 15.67
+  end
+
+  it "gives an order confirmation" do
+    expect(takeaway.confirmation).to be_an_instance_of(String)
```

Comment:
I just learned this morning that you can use the RSpec Output Matcher to test to see if a certain string is returned by your method. For example:

```
it "gives an order confirmation" do
    expect do 
      takeaway.confirmation
    end.to_output("Whatever string you want here").to_stdout
end
```